src/frontend/src/btc/derived/btc-pending-sent-transactions-status.derived.ts

```diff
@@ -9,9 +9,16 @@ import { testnets } from '$lib/derived/testnets.derived';
 import { nonNullish } from '@dfinity/utils';
 import { derived, type Readable } from 'svelte/store';
 
-export const initHasPendingSentTransactions = (
+export enum BtcPendingSentTransactionsStatus {
+	PRESENT = 'has-pending-transactions',
+	EMPTY = 'empty-pending-transactions',
```

Comment:
Instead of PRESENT and EMPTY, I was asking myself if we should named them SOME and NONE as in a Rust optional type. Just thought about sharing if interesting, I'm fine with PRESENT and EMPTY.

Reply:
Oh, sorry. I didn't see the comment, my bad. I saw just "LGTM, thx" and I assumed I could merge already.

I'll change in another PR.

Reply:
Ah no no, that was just an idea that crossed my mind. As I said, I'm fine with the current naming.